dashboard/app/views/shared/_course_tall_block.haml

```diff
@@ -30,7 +30,7 @@
   - title = t('upsell.unplugged_conditionals.title')
   - body = t('upsell.unplugged_conditionals.body')
 
-- elsif [Script::COURSE1_NAME, Script::COURSE2_NAME,Script::COURSE3_NAME, Script::COURSE4_NAME].include? id
+- elsif [Script::COURSE1_NAME, Script::COURSE2_NAME,Script::COURSE3_NAME, Script::COURSE4_NAME, Script::COURSEA_NAME, Script::COURSEB_NAME, Script::COURSEC_NAME, Script::COURSED_NAME, Script::COURSEE_NAME, Script::COURSEF_NAME].include? id
```

Comment:
I wonder if this check should belong in a model or helper somewhere? What determines whether or not a script belongs in this list?